docs/source/http-mode.md

```diff
@@ -1,8 +1,6 @@
 ## `notebook-http` Mode
 
-**NOTE** This mode may one day become a plug-in to the Jupyter Kernel Gateway or split into a separate project. The concepts documented here, however, will survive any move.
-
-The `KernelGatewayApp.api` command line argument can be set to `notebook-http`. In this mode, the kernel gateway exposes annotated cells in the `KernelGatewayApp.seed_uri` notebook as HTTP resources.
+The `KernelGatewayApp.api` command line argument can be set to `kernel_gateway.notebook_http`. This personality has the kernel gateway expose annotated cells in the `KernelGatewayApp.seed_uri` notebook as HTTP resources.
```

Comment:
Maybe the indicate that "personality" and "mode" are synonymous:

"This mode, or *personality*, has the kernel gateway expose annotated cells in the `KernelGatewayApp.seed_uri` notebook as HTTP resources."